Search-algorithms/src/base/QuicksortV1.java

```diff
@@ -0,0 +1,52 @@
+package base;
+
+public class QuicksortV1 extends SorterAlgorithm{
+	
+	private void swap(Integer a,Integer b){
+		Integer tmp;
+		tmp=a;
+		a=b;
+		b=tmp;
+	}
+	
+	private Integer partition(Integer[] arrayToBeOrdered, Integer firstElement, Integer last){
+		
+		Integer pivot=arrayToBeOrdered[last-1];
+		Integer pivotIndex=firstElement-1;
+		
+		for(Integer j=firstElement;j<=last-1;j++) {
+			if(arrayToBeOrdered[j]<pivot){
+				pivotIndex=pivotIndex+1;
+				swap(arrayToBeOrdered[pivotIndex], arrayToBeOrdered[j]);
+			}else
+			swap(arrayToBeOrdered[pivotIndex], arrayToBeOrdered[arrayToBeOrdered.length-1]);
+		}
+		return pivotIndex;
+	}
+	
+
+	@Override
+	public Integer[] sorting(Integer[] arrayToBeOrdered, Integer firstElement, Integer lastElement) {
```

Comment:
non ordinerai niente con questo metodo, il vettore ordinato viene restituito in output